content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -342,36 +388,40 @@ kubectl get service hostnames -o json
 }
 ```
 
-* Is the port you are trying to access in `spec.ports[]`?  
-* Is the `targetPort` correct for your `Pods` (many `Pods` choose to use a different port than the `Service`)?
-* If you meant it to be a numeric port, is it a number (9376) or a
-string "9376"?
-* If you meant it to be a named port, do your `Pods` expose a port
-with the same name?
-* Is the port's `protocol` the same as the `Pod`'s?
+* Is the `Service` port you are trying to access listed in `spec.ports[]`?
+* Is the `targetPort` correct for your `Pods` (some `Pods` use a different port than the `Service`)?
+* If you meant to use a numeric port, is it a number (9376) or a string "9376"?
+* If you meant to use a named port, do your `Pods` expose a port with the same name?
+* Is the port's `protocol` correct for your `Pods`?
 
 ## Does the Service have any Endpoints?
```

Comment:
Is it worth mentioning EndpointSlice here? I'm thinking maybe not until that's GA.

Reply:
agreed